lib/go-parser/index.ts

```diff
@@ -151,23 +154,26 @@ export async function goModulesToScannedProjects(
       await eventLoopSpinner.spin();
     }
 
-    const depGraph = await new GoBinary(goBinary).depGraph();
+    try {
+      const depGraph = await new GoBinary(goBinary).depGraph();
+      if (!depGraph) {
+        continue;
+      }
 
-    if (!depGraph) {
-      continue;
+      const depGraphFact: DepGraphFact = {
+        type: "depGraph",
+        data: depGraph,
+      };
+      scanResults.push({
+        facts: [depGraphFact],
+        identity: {
+          type: DEP_GRAPH_TYPE,
+          targetFile: filePath,
+        },
+      });
+    } catch (err) {
+      debug(`Go binary scan for file ${filePath} failed: ${err}`);
```

Comment:
Would it make sense to add some test to check this?

Reply:
+1

Reply:
I could use a binary that we know fails, but looking at those, the smallest one is 7MB...and I'd need to check that into the git repo.

Reply:
I don't mind adding a test in another PR.